frontend/hub/namespaces/hooks/useHubNamespaceActions.tsx

```diff
@@ -11,18 +11,30 @@ import {
 import { HubRoute } from '../../main/HubRoutes';
 import { HubNamespace } from '../HubNamespace';
 import { useDeleteHubNamespaces } from './useDeleteHubNamespaces';
+import { useWindowLocation } from '../../../../framework/components/useWindowLocation';
+import { useSignAllCollections } from '../../collections/hooks/useSignAllCollections';
+import { useHubContext } from '../../common/useHubContext';
 
 export function useHubNamespaceActions(options?: {
   onHubNamespacesDeleted: (namespaces: HubNamespace[]) => void;
+  onHubNamespacesSignAllCollections?: (namespaces: HubNamespace) => void;
+  isDetailsPageAction?: boolean;
 }) {
   if (!options) {
-    options = { onHubNamespacesDeleted: () => {} };
+    options = { onHubNamespacesDeleted: () => {}, onHubNamespacesSignAllCollections: () => {} };
   }
   const { t } = useTranslation();
+  const { settings, featureFlags } = useHubContext();
+  const signing_service = settings.GALAXY_COLLECTION_SIGNING_SERVICE;
+  const can_upload_signatures = featureFlags.can_upload_signatures;
```

Comment:
This is related to Insights mode. cc @himdel for details

Reply:
It should be `can_create_signatures`, as in https://github.com/ansible/ansible-hub-ui/blob/master/src/utilities/can-sign.ts#L7 :)

Reply:
Ooh... the original is `canSign && !can_upload_signatures`. So `!can_upload_signatures` is right :+1:
It's just missing that `canSign` part, which boils to that `can_create_signatures` (+ permissions)